### ISSUE TITLE
[Solved] Server error when trying to login with the desktop linux snap app (Yunohost server)

### ISSUE THREAD

**Issue by raphraph** [invoiceninja/admin-portal#431]
**invoiceninja version:** 5.3.60~ynh1

When trying to login using the desktop linux snap app (v5.0.77) I get the following error:

`Error: please check that Invoice Ninja v5 is installed on the server`

Can someone guide me on solving this error ? or can someone tell me if there are logs anywhere where I could find more information about the error ?

I have a running invoiceninja 5 on my yunohost, and ports (443 among others) are opened. I need more information to understand if it's a config error on my side, or an app error.

Thanks !

**Comment by hillelcoren** [comment]
Hi,

I'm not sure, the most likely reason for the error is using the wrong value for the URL

**Comment by raphraph** [comment]
Thanks but this is something I already checked and re-checked.

I can correctly access the login page in a web browser with this URL : `https://invoiceninja.mydomainname.com` and I tried copy-pasting this URL in the app login form... with no success.

The error message is about Invoice Ninja "version" since it asks to check that it is v5 that is installed. Is this just a "bad" error message, or can it help understand what's wrong ? The version installed on the server is `v5.3.60~ynh1` (from Yunohost).

**Comment by hillelcoren** [comment]
The error message is shown if the response from the server doesn't include a x-app-version header, maybe you need to add /public to the URL?

**Comment by raphraph** [comment]
Thanks for the input, I just solved the problem.

The x-app-version was included in the header response allright.
But when I tested in an isolated tab I realized that when accessing the URL I was redirected to my Yunohost instance login page first, and had to login before being able to reach the invoiceninja URL.
The problem was that I did not authorize the app (in the Yunohost settings) to be "public" and therefore it was not accessible from an external client.
